experimental/host/mt7688/gpio.go

```diff
@@ -10,53 +10,53 @@ import (
 
 // All the pins supported by the CPU.
 var (
-	GPIO0  *Pin // I2S_SDI
-	GPIO1  *Pin // I2S_SDO
+	GPIO0  *Pin // I2S_DIN
+	GPIO1  *Pin // I2S_DOUT
 	GPIO2  *Pin // I2S_WS
-	GPIO3  *Pin // I2S_CLK
-	GPIO4  *Pin // I2C_SCLK
-	GPIO5  *Pin // I2C_SD
-	GPIO6  *Pin // SPI_CS1
+	GPIO3  *Pin // I2S_SCK
+	GPIO4  *Pin // I2C_SCL
+	GPIO5  *Pin // I2C_SDA
+	GPIO6  *Pin // SPI_CS1, CLK0
 	GPIO7  *Pin // SPI_CLK
 	GPIO8  *Pin // SPI_MOSI
 	GPIO9  *Pin // SPI_MISO
 	GPIO10 *Pin // SPI_CS0
-	GPIO11 *Pin // GPIO0
-	GPIO12 *Pin // UART_TXD0
-	GPIO13 *Pin // UART_RXD0
-	GPIO14 *Pin // MDI_TP_P1
-	GPIO15 *Pin // MDI_TN_P1
-	GPIO16 *Pin // MDI_RP_P1
-	GPIO17 *Pin // MDI_RN_P1
-	GPIO18 *Pin // MDI_RP_P2
-	GPIO19 *Pin // MDI_RN_P2
-	GPIO20 *Pin // MDI_TP_P2
-	GPIO21 *Pin // MDI_TN_P2
-	GPIO22 *Pin // MDI_TP_P3
-	GPIO23 *Pin // MDI_TN_P3
-	GPIO24 *Pin // MDI_RP_P3
-	GPIO25 *Pin // MDI_RN_P3
-	GPIO26 *Pin // MDI_RP_P4
-	GPIO27 *Pin // MDI_RN_P4
-	GPIO28 *Pin // MDI_TP_P4
-	GPIO29 *Pin // MDI_TN_P4
-	GPIO30 *Pin // EPHY_LED4_N_JTRST_N (7688KN)
-	GPIO31 *Pin // EPHY_LED3_N_JTCLK (7688KN)
-	GPIO32 *Pin // EPHY_LED2_N_JTMS (7688KN)
-	GPIO33 *Pin // EPHY_LED1_N_JTDI (7688KN)
-	GPIO34 *Pin // EPHY_LED0_N_JTDO (7688KN)
-	GPIO35 *Pin // WLED_N (7688KN)
-	GPIO36 *Pin // PERST_N
-	GPIO37 *Pin // REF_CLKO
-	GPIO38 *Pin // WDT_RST_N
-	GPIO39 *Pin // EPHY_LED4_N_JTRST_N (7688AN)
-	GPIO40 *Pin // EPHY_LED3_N_JTCLK (7688AN)
-	GPIO41 *Pin // EPHY_LED2_N_JTMS (7688AN)
-	GPIO42 *Pin // EPHY_LED1_N_JTDI (7688AN)
-	GPIO43 *Pin // EPHY_LED0_N_JTDO (7688AN)
-	GPIO44 *Pin // WLED_N (7688AN)
-	GPIO45 *Pin // UART_TXD1
-	GPIO46 *Pin // UART_RXD1
+	GPIO11 *Pin // CLK0
+	GPIO12 *Pin // UART0_TX
+	GPIO13 *Pin // UART0_RX
+	GPIO14 *Pin // PWM0
+	GPIO15 *Pin // PWM1
+	GPIO16 *Pin // UART2_TX
+	GPIO17 *Pin // UART2_RX
+	GPIO18 *Pin // PWM0
+	GPIO19 *Pin // PWM1
+	GPIO20 *Pin // UART2_TX, PWM2
+	GPIO21 *Pin // UART2_RX, PWM3
+	GPIO22 *Pin
+	GPIO23 *Pin
+	GPIO24 *Pin
+	GPIO25 *Pin
+	GPIO26 *Pin
+	GPIO27 *Pin
+	GPIO28 *Pin
+	GPIO29 *Pin
+	GPIO30 *Pin // JTAG_TRST
```

Comment:
JTAG0_TRST, and similar below.

It likely never happens for a board to have multiple jtag ports, but I prefer to use consistent naming convention.

Reply:
Ah, I notice the pins for JTAG differ based on chipset (AN/KN)

Should I mark them out in the comments, like this?
```go
	GPIO30 *Pin // JTAG0_TRST (7688KN)
	GPIO31 *Pin // JTAG0_TCK  (7688KN)
	GPIO32 *Pin // JTAG0_TMS  (7688KN)
	GPIO33 *Pin // JTAG0_TDI  (7688KN)
	GPIO34 *Pin // JTAG0_TDO  (7688KN)
	...
	GPIO39 *Pin // JTAG0_TRST (7688AN)
	GPIO40 *Pin // JTAG0_TCK  (7688AN)
	GPIO41 *Pin // JTAG0_TMS  (7688AN)
	GPIO42 *Pin // JTAG0_TDI  (7688AN)
	GPIO43 *Pin // JTAG0_TDO  (7688AN)
```

Or make this package more specific to AN? (which seems to be _the_ chip used) and remove references to KN